gateway-js/src/__tests__/gateway/endToEnd.test.ts

```diff
@@ -8,9 +8,10 @@ import { GraphQLSchemaModule } from '@apollo/subgraph/src/schema-helper';
 import { buildSchema, ObjectType, ServiceDefinition } from '@apollo/federation-internals';
 import gql from 'graphql-tag';
 import { printSchema } from 'graphql';
-import LRUCache from 'lru-cache';
```

Comment:
There are no more usages of `lru-cache` in the gateway package. Can you remove it from the dependencies list in `gateway-js/package.json`? Then update the top-level `package-lock.json` by running `npm install`.